Scripts/script-AquatoneDiscover.yml

```diff
@@ -0,0 +1,34 @@
+commonfields:
+  id: AquatoneDiscover
+  version: -1
+name: AquatoneDiscover
+script: |-
+  import subprocess
+  import time
+
+  domain = demisto.args()['domain']
+  res = subprocess.check_output(['aquatone-discover','--domain',domain])
```

Comment:
If the process fails and the exit status is 1, then we get
![image](https://user-images.githubusercontent.com/7270217/39266313-5bce55c6-48d2-11e8-9566-669bd3acd249.png)

I think we should better handle this and print the original output